pages/stylesheets/all.scss

```diff
@@ -29,4 +29,28 @@ margin-bottom: 5px;
 .ResourceCard_resource__-Szc9 .ResourceCard_tags__container__3A8cx {
   margin-bottom: 0px;
   display: inherit;
+}
+
+.convPrompt li {
+  background: url('/assets/images/convPromptIcon.png') no-repeat left top;
+  padding: 0px 0px 10px 45px;
+  list-style: none;
+  margin-bottom: 25px;
+}
+
+.convPromptText {
+  font-weight:600;
+  margin: 0px;
+}
+
+.convSupportText {
+  font-size: 16px !important;
+  font-size: 1rem !important;
+  line-height: 1.25 !important;
+  color:#505a5f;
+}
+
+.convSupportText a {
+  color: #025ea6;
+  font-weight:600;
```

Comment:
Can we use `hyphen-cased` class names for consistency with the design system classes (and the others in this file)?

(I didn't catch this on the last PR, sorry)